nextgen/config/src/project/task_config.rs

```diff
@@ -97,6 +97,8 @@ cacheable!(
         pub env: FxHashMap<String, String>,
 
         pub target: Target,
+
+        pub skip_if_missing: Option<bool>,
```

Comment:
Let's just call it `optional`. Not sure skip if missing would make sense to end users.

Reply:
agreed: https://github.com/moonrepo/moon/pull/1276/commits/05b29703d4f7da515f9b0af8391f0ad7d6ed31fb